app-conf/FetcherConf.xml

```diff
@@ -36,6 +36,21 @@
       <sampling_enabled>false</sampling_enabled>
     </params>
   </fetcher>
+  <!--
+    This is an experimental replacement for the MapReduceFetcherHadoop2 that attempts to burn
+    through queues of jobs faster by pulling data directly from HDFS rather than going through the
+    job history server.
+  -->
+  <!--
+  <fetcher>
+    <applicationtype>mapreduce</applicationtype>
+    <classname>com.linkedin.drelephant.mapreduce.MapReduceFSFetcherHadoop2</classname>
```

Comment:
Can you package it inside com.linkedin.drelephant.mapreduce.fetchers.MapReduceFSFetcherHadoop2?

Create a ```fetchers``` directory under ```mapreduce``` and move this and the old fetcher inside it.